tests/fast-integration/eventing-test/eventing-test-prep.js

```diff
@@ -112,9 +111,9 @@ describe('Eventing tests preparation', function() {
     );
   }
 
-  after(async function() {
+  afterEach(async function() {
     // if the test preparation failed, perform the cleanup
-    if (this.currentTest.err && this.currentTest.err.code === fatalErrCode) {
+    if (this.currentTest.state === 'failed') {
```

Comment:
Why this is in `afterEach`? Should it be in `after` where we get the overall test results?